test/runtests.jl

```diff
@@ -352,10 +352,6 @@ end
 end
 
 @testset "MPI" begin
-    if Sys.iswindows()
-        @test LAMMPS.API.lammps_config_has_mpi_support() == 0
-    else
-        @test LAMMPS.API.lammps_config_has_mpi_support() != 0
-        @test success(pipeline(`$(MPI.mpiexec()) -n 2 $(Base.julia_cmd()) mpitest.jl`, stderr=stderr, stdout=stdout))
-    end
+    @test LAMMPS.API.lammps_config_has_mpi_support() != 0
+    @test success(pipeline(`$(MPI.mpiexec()) -n 2 $(Base.julia_cmd()) mpitest.jl`, stderr=stderr, stdout=stdout))
```

Comment:
Sorry, I meant, make the `mpitest.jl` fail if it is run without MPI support. In my earlier hybris, I trusted this test to not work without MPI support, but it would just happily run with LAMMPS not using MPI.